scripts/west_commands/sign.py

```diff
@@ -446,9 +446,9 @@ def preprocess_toml(self, config_dir, toml_basename, subdir):
         self.command.inf(quote_sh_list(preproc_cmd))
         subprocess.run(preproc_cmd, check=True, cwd=self.build_dir)
 
-    def sign(self, command, build_dir, build_conf, formats):
-        self.command = command
-        args = command.args
+    def sign(self, cmd, build_dir, build_conf, formats):
+        self.command = cmd
```

Comment:
This rename doubles the size of the change in this function. I agree the shorter `cmd` is long enough and better but I don't think it's so much better that it deserves that much git churn and the resulting cherry-pick/merge/rebase conflicts.